Capfile

```diff
@@ -24,8 +24,8 @@ require "capistrano/rails/migrations"
 require "capistrano/puma"
 require "capistrano/rails/console"
 require "capistrano/scm/git"
-require 'capistrano/sidekiq'
-require 'capistrano/sidekiq/monit'
+# require 'capistrano/sidekiq'
```

Comment:
Is there a reason to keep these lines? It'll always be difficult to reason about why they weren't removed when someone else read the code in the future  :)

Reply:
Thanks @wenderjean, I agree. I'll remove these comments 👍🏻